docs/4.3/cli-docs.md

```diff
@@ -602,7 +609,7 @@ Create an invitation token
 
 | Name | Default Value(s) | Allowed Value(s) | Description
 |------|---------|----------------|----------------------------|
-| `--type` | none | `trusted_cluster` , `node` , `signup` <!--TODO Confirm this.are any other types valid in /Users/heather/teleport/roles.go--> | Type of token to add
+| `--type` | none | `trusted_cluster` , `node`   | Type of token to add
```

Comment:
The `proxy` type should probably be listed here as well.

Reply:
ty, fixed